src/components/RightPane/SectionTable/SectionTableButtons.tsx

```diff
@@ -90,6 +90,20 @@ export const ScheduleAddCell = withStyles(styles)((props: ScheduleAddCellProps)
         }
     };
 
+    const closeCopyAndAlert = () => {
+        const url = new URL(document.location.href)
```

Comment:
```suggestion
        const url = new URL(window.location.href)
```
I generally prefer using window instead of document; even more so in React because there isn't even a document (i.e. because it's virtual)